### DISCUSSION TITLE
[Feature request] Make input textarea more bigger

### DISCUSSION THREAD

**Discussion by DanRotaru** [character-map-uwp/Character-Map-UWP#209] [Ideas]
This app is amazing, but it have small UI issues, font preview textarea is too small, with bigger font size, and I can't understand how font looks for specific text (if text is longer, some phrase or slogan)
So, just add possibility to:
1. Change textarea font size;
2. Change textarea size, or maybe instead letter preview put textarea view, and let user choose what he need, I think for more people to preview text is more important than just a some letter.

![image](https://user-images.githubusercontent.com/7759507/174470049-cbbbc289-99b9-49fe-8de1-779d3bead781.png)

Thank you.

**Comment by JohnnyWestlake** [comment]
Thanks for the feedback! There are a few options here that might suit your case:
![image](https://user-images.githubusercontent.com/2319473/174582526-cd079fbb-c31b-4fb0-a2c4-f50c81c180d9.png)

1 - You can hide the letter preview area entirely by clicking this button in the "..." menu
2 - You can resize the the area using the column dragger
3 - You can switch to the type ramp view using this button, which might be more appropriate for you
4 - You can hide the font list using this button. (You can reshow it by clicking the menu button in the top left to make it open as an overlay, and press button 4 again to lock it open)

Type Ramp View: 
![image](https://user-images.githubusercontent.com/2319473/174581908-c53ff1d4-f395-4f72-ae5c-83a2623d6333.png)

If none of these work for you I can consider adding something else, let me know what you think.
